src/fu-tool.c

```diff
@@ -3880,6 +3880,22 @@ main(int argc, char *argv[])
 		return EXIT_FAILURE;
 	}
 
+	/* show traceback */
+	if (g_getenv("FWUPD_VERBOSE") != NULL) {
+		guint64 threshold_ms = 1;
+		g_autofree gchar *str = NULL;
+		if (!fu_common_strtoull_full(g_getenv("FWUPD_VERBOSE"),
```

Comment:
this new behavior for `FWUPD_VERBOSE` needs to be documented, and also why is it it inconsistent with the daemon?

Reply:
I wasn't sure which made more sense; I started with the env var, then moved it to a daemon config option, and now I'm thinking the env var makes more sense for both.